sqlmesh/core/dialect.py

```diff
@@ -604,7 +604,7 @@ def format_model_expressions(
     Returns:
         A string representing the formatted model.
     """
-    if len(expressions) == 1:
+    if len(expressions) == 1 and isinstance(expressions[0], (Audit, Metric, Model)):
```

Comment:
Looks like something that should be a helper function. Provided its usage in `context.py`.

Reply:
Thanks for the suggestion, refactored it into an `is_meta_expression` helper under `dialect.py`.